<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveItem</class>
 <widget class="QWidget" name="SaveItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SaveItem</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,9">
   <item>
    <widget class="QLabel" name="imgLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>90</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="saveNameLabel">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>-1</pointsize>
         <weight>62</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size: 25px;
font-weight: 500;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lastModifiedLabel">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #cccccc;
font-size:0.6em;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
